translations.yml

```diff
@@ -63,6 +63,16 @@ parts:
       title: "Description for the visited link color setting"
       screenshot: "https://drive.google.com/file/d/1ZOoBb6K_mpYtyd3ulkEKawmSj1dZPuFw/view?usp=sharing"
       value: "Text color for visited link elements"
+  - translation:
+      key: "txt.help_center_copenhagen_theme.hover_link_color_label"
+      title: "Label for the hover link color setting"
+      screenshot: "https://drive.google.com/file/d/1E_5jIIAasv6-wC3YM-G_MDHB04_PKw66/view?usp=sharing"
```

Comment:
Can we have a screenshot that shows the string in use?
Just rendering the identifier is probably confusing for the translators. The screenshot is mean to show how the UI is going to look like with the string bring extracted.